src/extension.ts

```diff
@@ -1,56 +1,143 @@
-import * as vscode from 'vscode';
-import * as path from 'path';
-import * as fs from 'fs';
+import * as vscode from "vscode";
```

Comment:
The different formatter is causing a an expanded diff but that's on me

Reply:
Anything you want me to do on this?